docs/tutorial.rst

```diff
@@ -36,7 +36,9 @@ In the installer, be sure to check the "Add Python to PATH" checkbox.
 Create Virtual Environment
 --------------------------
 
-A Python Virtual Environment (virtualenv) is an isolated Python environment where you can install packages without modifying the system Python.  Using a virtualenv for cumulusci is recommended to avoid issues and conflicts with other applications using your system Python.
+A Python Virtual Environment (virtualenv) is an isolated Python environment where you can install packages without modifying the system Python.  Using a virtualenv for CumulusCI is recommended to avoid issues and conflicts with other applications using your system Python.
+
+For more information about virtual environments in Python, see the Python Packaging User Guide: https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments
```

Comment:
I moved this out of the macOS section because I feel like a Windows user might just skip over this and not see it. It applies to all users.